SPECS/safeclib/safeclib.spec

```
@@ -0,0 +1,72 @@
+Summary:        C11 Annex K functions
+Name:           libsafec
+Version:        3.7.1
+Release:        1%{?dist}
+License:        MIT
+URL:            https://github.com/rurban/safeclib
+Source0:        https://github.com/rurban/safeclib/releases/download/v3.7.1/safeclib-%{version}.tar.xz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  gcc
+BuildRequires:  make
+%if %{with_check}
+BuildRequires:  tzdata
+%endif
+
+%description
+Extension library providing C11 Annex K functions
+
+%package -n libsafec-devel
+Summary:        Development packages for libsafec
+Requires:       libsafec = %{version}-%{release}
+
+%description -n libsafec-devel
+Development files for libsafec
+
+%package -n libsafec-check
+Summary:        Tools to detect use of unsafe libc APIs
+
+%description -n libsafec-check
+Tools to detect use of unsafe libc APIs
```

The `-n` is not needed in this case and we tend to only add it, if the package name's prefix is different from the value of the `Name` tag:
```suggestion
%package devel
Summary:        Development packages for libsafec
Requires:       %{name} = %{version}-%{release}

%description devel
Development files for libsafec

%package check
Summary:        Tools to detect use of unsafe libc APIs

%description check
Tools to detect use of unsafe libc APIs
```